sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -234,7 +237,7 @@ def _buffered_send(self, events, **kwargs):
                 max_wait_time=self._max_wait_time,
                 max_buffer_length=self._max_buffer_length,
                 executor=self._executor,
-                max_worker=self._max_worker
+                max_worker=self._max_worker,
```

Comment:
Looks like passing in max_worker will always be None - so we could probably remove this parameter entirely?